### ISSUE TITLE
Normalization docs are unclear

### ISSUE THREAD

**Issue by jveitchmichaelis** [albumentations-team/albumentations#302]
## 📚 Documentation

I was preparing some data using Normalize and was getting weird results until I noticed the `max_pixel_value`. The docs say:

> Divide pixel values by 255 = 2**8 - 1, subtract mean per channel and divide by std per channel.

This is the code:

```
def normalize(img, mean, std, max_pixel_value=255.0):
    mean = np.array(mean, dtype=np.float32)
    mean *= max_pixel_value

    std = np.array(std, dtype=np.float32)
    std *= max_pixel_value

    denominator = np.reciprocal(std, dtype=np.float32)

    img = img.astype(np.float32)
    img -= mean
    img *= denominator
    return img
```

The confusion arises because it's not obvious from the docstring what data are assumed to be normalised. For example, I tried passing in 16-bit data so I used 65535 as my max pixel value with my mean and standard deviation in data units (not normalised). This gave weird results. I ended up using `max_pixel_number=1`.

So perhaps this can be clarified? I suggest:

> Normalizes an image by subtracting (mean *  max_pixel_value) and dividing by (std *  max_pixel_value), per channel. `max_pixel_value` should be 255 for 8-bit images.

**Comment by sezarxray** [comment]
Divide pixel values by 255 = 2**8 - 1   never apply!!!!!

**Comment by Dipet** [comment]
Docstring is fixed in #919
